src/main/java/nl/tudelft/planningstool/api/v1/UserOccurrenceAPI.java

```diff
@@ -53,6 +84,10 @@
                         .filter(o -> o.getStart_time() >= timeSlot.getStart())
                         .filter(o -> o.getEnd_time() <= timeSlot.getEnd())
                         .map(CourseOccurrenceResponse::from)
+                        .map(o -> {
+                            map.put(o.getCourse().getEdition(), null);
```

Comment:
Oops, this should be the same as https://github.com/Mentoraat/plannings-tool-server/pull/10/files#diff-7d3d669349095100287747d69d0c800aR73 #todo